tls_assets.go

```diff
@@ -70,12 +72,15 @@ type AssetsBundle map[AssetsBundleKey][]byte
 
 // ClusterComponents is a slice enumerating all the components that make up the
 // cluster.
+//
+// TODO add FlanneldComponent as soon as all guest clusters have Flanneld
+// certificates.
```

Comment:
I want to do this later.

Reply:
Why adding now the component?

Reply:
The issue this work tries to solve in the end is to provide a separate key pair to the flannel daemons used in each guest cluster. This here is the very first step to get the reference for the component used to generate certificates. We can do it in a smooth way and just wait until all guest clusters have the key pair generated. Then we can actually use them and add the new component to the lookup when reconciling guest clusters.